charts/cass-operator-chart/templates/customresourcedefinition.yaml

```diff
@@ -5946,6 +5946,276 @@ spec:
                     description: Deprecated. Use nodeAffinityLabels instead. Zone
                       name to pin the rack, using node affinity
                     type: string
+                  storageConfig:
```

Comment:
So you're effectively declaring `storageConfig` twice? Can't you do:

```yaml
storageConfig:
  description: storage config to use for the rack, overrides default DC storage config
  type: storageConfig
```

Reply:
@tomer I don't know if it is feasible
If so it is better!